bugbug/phabricator.py

```diff
@@ -291,3 +292,18 @@ def get_pending_review_time(rev: RevisionDict) -> timedelta | None:
         return datetime.utcnow() - last_exclusion_end_date
     else:
         return datetime.utcnow() - creation_date
+
+
+def fetch_diff_from_url(
+    revision_id, first_patch, second_patch=None, single_patch=False
+):
+    if single_patch:
+        url = f"https://phabricator.services.mozilla.com/D{revision_id}?id={first_patch}&download=true"
+    else:
+        url = f"https://phabricator.services.mozilla.com/D{revision_id}?vs={first_patch}&id={second_patch}&download=true"
+
+    response = requests.get(url)
+    if response.status_code == 200:
+        return response.text
+    else:
```

Comment:
Using `else` is not necessary here. I would flip it and make the exception conditional. 

Note: In general, it is better to keep the regular code flow more to the lift and the irregular logic to the right.

Reply:
c72504df8d3adc89c3b0824f6efbb075a2a92ce7